charts/bindplane/values.schema.json

```diff
@@ -0,0 +1,19 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "type": "object",
+    "properties": {
+      "nats": {
+        "type": "object",
+        "properties": {
+          "deploymentType": {
+            "type": "string",
+            "enum": ["StatefulSet", "Deployment"],
+            "description": "The type of NATS deployment, either StatefulSet or Deployment."
+          }
+        },
+        "required": ["deploymentType"],
+        "additionalProperties": true
+      }
+    }
+  }
+  
```

Comment:
This ensures the user can only configure StatefulSet or Deployment for `nats.deploymentType`. Basically making it an enum.